src/functions/template-tags/organizer.php

```diff
@@ -332,19 +347,25 @@ function tribe_get_organizer_website_url( $postId = null ) {
 	function tribe_get_organizer_website_link( $post_id = null, $label = null ) {
 		$post_id = tribe_get_organizer_id( $post_id );
 		$url     = tribe_get_event_meta( $post_id, '_OrganizerWebsite', true );
+		$target = apply_filters( 'tribe_get_event_organizer_link_target', '_self' );
+		$rel    = ( '_blank' === $target  ) ? 'noopener noreferrer' : 'external';
+		$label = apply_filters( 'tribe_get_organizer_website_link_label', $label );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 3 spaces but found 1 space

<sub>hash: c446801cfad2c1a122f24690361d2c48</sub>